.travis.yml

```diff
@@ -6,18 +6,21 @@ cache:
   directories:
     - $HOME/.composer/cache
 
-php:
-  - '7.0'
+matrix:
+  include:
+    - php: '7.0'
+      env: PREFER_LOWEST=""
+    - php: '7.1'
+      env: PREFER_LOWEST="--prefer-lowest"
 
 before_install:
-  - phpenv config-rm xdebug.ini
   - phpunit --self-update
   - npm install -g swagger-cli
 
 install:
-  - composer update --no-interaction --prefer-dist
+  - composer update --no-interaction --prefer-dist $PREFER_LOWEST
 
 script:
-  - vendor/bin/phpunit
```

Comment:
if we are removing phpunit because it's a project dependency and not a dev dependency then the same should be done for behat